app/views/asns/show.html.erb

```diff
@@ -54,9 +54,19 @@
   </div>
 </div>
 
-<%= link_to 'Back to ASN list',
-            data_centers_url(network: params[:network], sort_by: 'asn'),
-            class: 'btn btn-sm btn-secondary mb-4' %>
+<%= link_to 'Filter by delinquent',
+    asn_url(network: params[:network], asn: params[:asn], filter_by: :delinquent),
+    class: "btn btn-sm btn-secondary mb-4 #{'active' if params[:filter_by] == 'delinquent'}" %>
```

Comment:
Please add 'mr-1' class for each button. In the future please try to keep styling consistent with the rest of the page.